functions.php

```diff
@@ -108,3 +111,27 @@ function bcgov_blocks_theme_enqueue_admin_scripts(): void {
 }
 
 add_action( 'admin_enqueue_scripts', 'bcgov_blocks_theme_enqueue_admin_scripts' );
+
+/**
+ * If no custom logo has been chosen, sets up a default logo.
+ *
+ * @param string $html Markup from original get_custom_logo call.
+ * @since 1.0.0
+ *
+ * @return string
+ */
+function get_custom_logo_callback( $html ) {
+    if ( has_custom_logo() ) {
+        return $html;
```

Comment:
curious if there is a custom logo, how does this work?  Is the block theme editing do some magic here.
I say this in a humbling way, trying to learn from your experience, as this is a bit in uncharted territory.

Reply:
Yes, the site-logo block uses get_custom_logo() internally.
We're taking advantage of that to inject a default markup when a logo has not been chosen by the site owner.
If the site owner has used the theme customizer to pick a logo, we get to just return that.

This way we can simply use the wp:site-logo  tag in the header.